.github/workflows/release-please.yml

```diff
@@ -0,0 +1,56 @@
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          release-type: go
+          package-name: exatorrent
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        if: ${{ steps.release.outputs.release_created }}
+
+      - name: Set up NodeJS
+        uses: actions/setup-node@v4
+        with:
+          node-version: "^18"
+          cache: "npm"
+          cache-dependency-path: "internal/web/package-lock.json"
+        if: ${{ steps.release.outputs.release_created }}
+
+      - name: Build Web UI
+        run: |
+          make web-ci
+        if: ${{ steps.release.outputs.release_created }}
+
+      - name: Build
+        uses: crazy-max/ghaction-xgo@v3
+        with:
+          xgo_version: latest
+          go_version: 1.21
+          dest: build
+          targets: linux/amd64,linux/arm64,linux/arm-7,linux/386
```

Comment:
this is from winbuntu lasted.. so in some linuxed **will not work due the binary will require glibc >= 2.36 by example i am not able to run this in my toshiba satellite that cannot handle linux modesetting**.. and in the i9 i will not run a server cos its shipped with macos with guarantee